.travis.yml

```diff
@@ -65,7 +65,7 @@ script:
   - testscript -v testscripts/mod_vendor.txt
 
   # Prepare to test the png example from dvyukov/go-fuzz-corpus.
-  - go get -v -d github.com/dvyukov/go-fuzz-corpus/png
+  - git clone --depth=1 https://github.com/thepudds/go-fuzz-corpus $GOPATH/src/github.com/dvyukov/go-fuzz-corpus
```

Comment:
Hi Dmitry 👋 thanks for the review! 

> Do we need some fix in go-fuzz-corpus as well?

The go-fuzz README likely needs to be updated regarding how to use go-fuzz-corpus (#317), and/or go-fuzz-corpus could become one or more modules, including to avoid https://github.com/dvyukov/go-fuzz/issues/317#issuecomment-824874463. That was also the root cause of one of the ~5 reasons this testing was failing. I was taking an initial stab at a PR for go-fuzz-corpus, as well as using a go version directive in a go-fuzz-corpus go.mod as a potential workaround for the double '//go:build' issue (#313).

That said, I then switched gears to working around the go-fuzz-corpus module issues within travis (by creating a local test module in the travis script), including to keep this PR more self-contained.

In any event, I should switch the travis git clone back to dvyukov/go-fuzz-corpus, and that was my intent for this PR. Sorry for missing that.